src/main/java/com/conversions/BinaryToHexadecimal.java

```diff
@@ -0,0 +1,42 @@
+package src.main.java.com.conversions;
+
+import java.util.HashMap;
+
+public class BinaryToHexadecimal {
+
+    /**
+     * This method converts a binary number to
+     * a hexadecimal number.
+     *
+     * @param binary The binary number
+     * @return The hexadecimal number
+     */
+
+    public String binToHex(int binary) {
```

Comment:
The binary number may have 32bits or 64bits, so it's  unsuitable to use `int` type.

Reply:
Resolved